test/mocks/transformMock.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
can you add file license here

Reply:
Also can you change this to .ts

Reply:
Yeah, I think you can do something like
```
export default {
   process() { ... }
   getCacheKey() { ... }
}
```

Also how is this mock working exactly? Couple comments at the top of the file could be helpful

Reply:
added a comment to explain how it works